### PR TITLE
BZ#1975622 - [OCPRHV] [Docs] "engine" and "Manager" not correctly used in OCP docs.

### PR DESCRIPTION
**Pull request by stoobie**
enterprise_4.9
enterprise_4.8
enterprise_4.7
enterprise_4.6

For versions 4.6+

https://bugzilla.redhat.com/show_bug.cgi?id=1975622

This PR adds a new attribute that evaluates to "Manager" for OCP Enterprise and "Engine" for OCP Origin.

I replaced all relevant occurrences of either "Manager" or "Engine" that I could find. Notice that not all occurences of "engine" were replaced, because there are instances of that word being used in certain contexts in RHV.

Direct link to doc preview:
https://deploy-preview-35508--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -1046,7 +1046,7 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Integer
 
 |`<machine-pool>.platform.ovirt.instanceTypeID`
-|Optional. An instance type UUID, such as `00000009-0009-0009-0009-0000000000f1`, which you can get from the `https://<engine-fqdn>/ovirt-engine/api/instancetypes` endpoint.
+|Optional. An instance type UUID, such as `00000009-0009-0009-0009-0000000000f1`, which you can get from the `https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api/instancetypes` endpoint.
```
Comment (mburke5678) [inline modules/installation-configuration-parameters.adoc:1049]: This one looks a bit odd in the preview with a capital letter in the variable:
` the https://<Manager-fqdn>/ovirt-engine/api/instancetypes endpoint.`

File: modules/installation-initializing.adoc [hunk 2]
```diff
@@ -290,7 +290,7 @@ For production {product-title} clusters on which you want to perform installatio
 +
 [source,terminal]
 ----
-https://<engine-fqdn>/ovirt-engine/api <1>
+https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api <1>
```
Comment (mburke5678) [inline modules/installation-initializing.adoc:293]: The variable didn't work as it is in a code block, I think. Is this what you expected:
`https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api `

And the call-out has `<engine-fqdn>`.

File: modules/installation-launching-installer.adoc [hunk 3]
```diff
@@ -336,7 +336,7 @@ https://ovirtlab.example.com/ovirt-engine/api
 ----
 endif::openshift-origin[]
 +
-.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to the Engine?`, answer `y` or `N`. If you answer `N`, you must install {product-title} in insecure mode.
+.. The installer automatically generates a CA certificate. For `Would you like to use the above certificate to connect to the {rh-virtualization-engine-name}?`, answer `y` or `N`. If you answer `N`, you must install {product-title} in insecure mode.
```
Comment (mburke5678) [inline modules/installation-launching-installer.adoc:339]: Is this supposed to appear in _[Installing a cluster on RHV with customizations](https://deploy-preview-35508--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations)_? I'm not seeing it in the preview or [live docs](https://docs.openshift.com/container-platform/4.8/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations). I see these steps in the default RHV install.

Comment (mburke5678) [reply modules/installation-launching-installer.adoc:339]: Does the code switch between `Engine` and `Manager`?

File: modules/installing-rhv-setting-up-ca-certificate.adoc [hunk 4]
```diff
@@ -16,16 +16,16 @@ Later, you provide the certificate to the installation program.
...
 
 . Use either of these two methods to download the CA certificate:
-** Go to the Manager's webpage, `\https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
+** Go to the {rh-virtualization-engine-name}'s webpage, `\https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
 ** Run the following command:
 +
 [source,terminal]
 ----
-$ curl -k 'https://<engine-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
+$ curl -k 'https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
```
Comment (mburke5678) [inline modules/installing-rhv-setting-up-ca-certificate.adoc:24]: Appears as: 
`curl -k 'https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  `

File: modules/installing-rhv-setting-up-ca-certificate.adoc [hunk 5]
```diff
@@ -16,16 +16,16 @@ Later, you provide the certificate to the installation program.
 .Procedure
 
 . Use either of these two methods to download the CA certificate:
-** Go to the Manager's webpage, `\https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
+** Go to the {rh-virtualization-engine-name}'s webpage, `\https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
```
Comment (mburke5678) [inline modules/installing-rhv-setting-up-ca-certificate.adoc:19]: Does this module need to appear in the default install and install with customizations? I am not seeing it. I see it in the restricted install

File: modules/installing-rhv-setting-up-installation-machine.adoc [hunk 6]
```diff
@@ -7,7 +7,7 @@
 = Setting up the installation machine
 
 
-To run the binary `openshift-install` installation program and Ansible scripts, set up the {rh-virtualization} Manager or an {op-system-base-full} computer with network access to the {rh-virtualization} environment and the REST API on the {rh-virtualization} Manager/oVirt Engine.
+To run the binary `openshift-install` installation program and Ansible scripts, set up the {rh-virtualization} Manager or an {op-system-base-full} computer with network access to the {rh-virtualization} environment and the REST API on the {rh-virtualization-engine-name}.
```
Comment (mburke5678) [inline modules/installing-rhv-setting-up-installation-machine.adoc:10]: Do you need to get this one: `{rh-virtualization} Manager`?

File: modules/installing-rhv-verifying-rhv-environment.adoc [hunk 7]
```diff
@@ -51,7 +51,7 @@ These requirements are based on the default resources the installation program u
 [source,terminal]
 ----
 $ curl -k -u <username>@<profile>:<password> \ <1>
-https://<engine-fqdn>/ovirt-engine/api <2>
+https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api <2>
```
Comment (mburke5678) [inline modules/installing-rhv-verifying-rhv-environment.adoc:54]: Appears as: `https://<{rh-virtualization-engine-name}-fqdn>/ovirt-engine/api `

**Comment by mburke5678** [comment]
@stoobie A few comments. In the install docs, sometimes you have:

the RHV Manager
RHV Manger
the Manager 

Is this acceptable?

**Comment by mburke5678** [comment]
FYI: @vrutkovs

File: modules/installation-configuration-parameters.adoc (hunk 1 above)
Comment (stoobie) [reply modules/installation-configuration-parameters.adoc:1049]: You're right. I'm changing all those instances back to "engine-fqdn".

File: modules/installation-initializing.adoc (hunk 2 above)
Comment (stoobie) [reply modules/installation-initializing.adoc:293]: Fixed. Changed back to engine-fqdn.

File: modules/installation-launching-installer.adoc (hunk 3 above)
Comment (stoobie) [reply modules/installation-launching-installer.adoc:339]: @mgold1234 Can you help here?

File: modules/installing-rhv-setting-up-ca-certificate.adoc (hunk 4 above)
Comment (stoobie) [reply modules/installing-rhv-setting-up-ca-certificate.adoc:24]: Changed to <engine-fqdn>

File: modules/installing-rhv-setting-up-ca-certificate.adoc (hunk 5 above)
Comment (stoobie) [reply modules/installing-rhv-setting-up-ca-certificate.adoc]: No, this should not appear there.

File: modules/installing-rhv-setting-up-installation-machine.adoc (hunk 6 above)
Comment (stoobie) [reply modules/installing-rhv-setting-up-installation-machine.adoc]: Yes, thanks for the catch. I just changed a bunch of those.

File: modules/installing-rhv-verifying-rhv-environment.adoc (hunk 7 above)
Comment (stoobie) [reply modules/installing-rhv-verifying-rhv-environment.adoc]: Fixed.

**Comment by stoobie** [comment]
@mburke5678 I made some fixes based on your comments. This has received QE approval. So if you still approve, this can be merged.

**Pull request merged**